### ISSUE TITLE
Bug Report: SQLite DateTime Property Not Loaded Correctly in Framework

### ISSUE THREAD

**Issue by NirmalKumarYuvaraj** [dotnet/maui#15701]
### Description

The DateTime property in SQLite is not loading properly within the framework. The issue occurs when retrieving values from the SQLite database, specifically related to DateTime properties. The loaded DateTime values do not match the expected values.

### Steps to Reproduce

1. Create a MAUI App.
2. Define a model class with at least one DateTime property.
3. Set up an SQLite database connection in the MAUI app.
4. Create a table in the SQLite database corresponding to the model class.
5. Insert sample data into the SQLite table.
6. Retrieve the data from the SQLite table and load it into the DateTime property of the model.
7. Verify that the loaded DateTime values in the model do not match the expected values or exhibit any other inconsistencies.

### Link to public reproduction project repository

https://github.com/NirmalKumarYuvaraj/SQLSample

### Version with bug

7.0.49

### Last version that worked well

6.0

### Affected platforms

Windows, I was *not* able test on other platforms

### Affected platform versions

Windows SDK 10.0.17134.0

**Comment by Eilon** [comment]
Hi @NirmalKumarYuvaraj can you provide more info on what exactly didn't match? Can you give an example of a datetime you entered, but came out differently? Maybe it's an issue with timezones, or UTC conversions, or something similar?

**Comment by drasticactions** [comment]
I don't believe this is a MAUI bug,

https://github.com/drasticactions/MauiRepros/tree/main/SqlSample

![image](https://github.com/dotnet/maui/assets/898335/e3f4ac11-062e-4fbc-a56a-e412c3c4228d)

You can reproduce this in a console app. The library you're using is `netstandard`, and the MAUI UI Framework has no bearing on its functions.

![image](https://github.com/dotnet/maui/assets/898335/140cd8c9-b792-4149-b402-1258367303c3)

Changing your code to insert the object directly instead of trying to query it works fine. Your SQL code is wrong.

**Comment by NirmalKumarYuvaraj** [comment]
I apologize for any confusion caused. Although it may appear that directly inserting an object is an easy solution, it is typically advised to employ queries when adding objects to a database. I must utilize an SQL query to input datetime values.

**Comment by Eilon** [comment]
@NirmalKumarYuvaraj is there still any issue then, or should we close this?

**Comment by NirmalKumarYuvaraj** [comment]
Indeed, there is a problem as I cannot pass the datetime value in the query.

**Comment by drasticactions** [comment]
@NirmalKumarYuvaraj The problem is your code. As far as I can tell, It has nothing to do with MAUI or code within this repo. I created an example in a .NET Console using your code, showing the same behavior.

Unless you can show that this only happens within the context of a MAUI application, I think we should close this issue.
